<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LightDialog</class>
 <widget class="QDialog" name="LightDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>285</width>
    <height>290</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>285</width>
    <height>233</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Light</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Источник света</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="positionGroupBox">
     <property name="title">
      <string>Позиция</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="xLabel">
        <property name="text">
         <string>X =</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="xLineEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="yLabel">
        <property name="text">
         <string>Y =</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="yLineEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="zLabel">
        <property name="text">
         <string>Z =</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="zLineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Интенсивность</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="intensityLabel">
        <property name="text">
         <string>I =</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="intensityLineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="addPushButton">
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Отменить</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
